### ISSUE TITLE
v-click on sublist

### ISSUE THREAD

**Issue by nicolasunivlr** [slidevjs/slidev#362]
**Describe the bug**
`<v-click> `doesn't work on sublist like 
- element1
  - subelement1
  - subelement2
- element2
  - ... 

**To Reproduce**
I tried :

```
<ul>
  <li v-click>
    lorem ipsum
    <ul>
      <li v-click>lorem ipsum</li>
      <li v-click>lorem ipsum</li>
      <li v-click>lorem ipsum</li>
      <li v-click>...</li>
    </ul>
  </li>
  <li v-click>lorem ipsum</li>
</ul>
```

also :
```
<v-clicks>

- lorem ipsum
  - sub lorem ipsum
  - sub lorem ipsum
  - sub lorem ipsum
  - sub ...
- lorem ipsum

</v-clicks>
```

I would like "sub lorem ipsum" appears step by step

**Desktop (please complete the following information):**
 - OS: linux
 - Browser: firefox
 - Slidev version: 0.25.8

**Comment by lwolodkiewiczftdev** [comment]
@nicolasunivlr 
as workaround you can do this this way: 

`
<ul>
  <li v-click="1">
    lorem ipsum
    <ul>
      <li v-click="2">lorem ipsum</li>
      <li v-click="3">lorem ipsum</li>
      <li v-click="4">lorem ipsum</li>
      <li v-click="5">...</li>
    </ul>
  </li>
  <li v-click="6">lorem ipsum</li>
</ul>
`

**Comment by Jak2k** [comment]
Isn't this issue closed by #557?
